frontend/src/components/notifications/NotificationList.tsx

```
@@ -0,0 +1,75 @@
+
+  return (
+    <div className="notification-list">
+      <button onClick={toggleDropdown} className='icon-button notification-button content-interactions'>
+        <BellIcon className='icon' />
+
+        {unreadCount > 0 && <span className="badge">{unreadCount}</span>}
+        {isOpen && (
+          <div className="notification-dropdown">
+            {notifications.length > 0 ? (
+              notifications.map(notification => (
+                <Notification notification={notification} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)